pkg/routeagent_driver/handlers/ovn/host_networking.go

```diff
@@ -115,8 +115,7 @@ func (ovn *Handler) programRulesForRemoteSubnets(subnets []string, ruleFunc func
 
 func (ovn *Handler) getNextHopOnK8sMgmtIntf() (*net.IP, error) {
 	link, err := ovn.netLink.LinkByName(OVNK8sMgmntIntfName)
-
-	if err != nil && !errors.Is(err, netlink.LinkNotFoundError{}) {
```

Comment:
Don't we want to check if err is not found here ?

Reply:
Just seeing your question... the `errors.Is(err, netlink.LinkNotFoundError{})` check was actually incorrect and always returned false. But that was actually a good thing b/c, if it wasn't found, we would try to use a nil `link` instance. So I don't think this needs to check for `LinkNotFound`.